bsd/api.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+import os
+from bsdapi.BsdApi import Factory as BsdApiFactory
+from django.conf import settings
+
+
+class Bsd:
+    api = BsdApiFactory().create(
+        id = settings.BSD_API_ID,
```

Comment:
According to PEP8 (https://www.python.org/dev/peps/pep-0008/) there should be no spaces around around the equals here, but this is an incredibly minor thing. Would just double-check that your IDE is properly linting for PEP8.

Reply:
What do you use to lint for PEP8?

Reply:
I use Atom with the Atom IDE integration (https://ide.atom.io/) and ide-python (https://atom.io/packages/ide-python) linter-ui-default (https://atom.io/packages/linter-ui-default).